contact.html

```diff
@@ -0,0 +1,49 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Bunrith Buth | Contact Me</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/meyer-reset/2.0/reset.css" />
+    <link rel="stylesheet" type="text/css" media="screen" href="assets/css/style.css" />
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
+</head>
+
+<body>
+    <header>
+        <div class="headercontent">
+            <a href="index.html">
+                <div class="headerlogo">
+                    <h1>Bunrith Buth</h1>
+                </div>
+            </a>
+            <div class="headerRightNav">
+                <a href="index.html">About</a>
+                <span> &nbsp;&nbsp;|&nbsp;&nbsp; </span>
+                <a href="portfolio.html">Portfolio</a>
+                <span> &nbsp;&nbsp;|&nbsp;&nbsp; </span>
+                <a href="contact.html">Contact</a>
+            </div>
+        </div>
+    </header>
+    <div id="mainFrame">
+        <div id="body">
+            <h1>Contact</h1>
+            <div class="bar"></div>
+            Name<br>
+            <input style="width: 100%" ="text" name="Name" placeholder="Enter Name"><br>
+            Email<br>
+            <input style="width: 100%" type="text" name="Email" placeholder="Enter Email"><br>
```

Comment:
try to keep your styling inside your css file so you don't have to search for it if you need to change it later :)